src/parser/typescript/analyze/index.ts

```diff
@@ -380,7 +398,7 @@ function analyzeRuneVariables(
       // See https://github.com/sveltejs/svelte/blob/41b5cd6f5daae3970a9927e062f42b6b62440d16/packages/svelte/types/index.d.ts#L2615
       case "$props": {
         // Use type parameters to avoid `@typescript-eslint/no-unsafe-assignment` errors.
-        appendDeclareFunctionVirtualScripts(globalName, ["<T>(): T"]);
+        appendDeclareFunctionVirtualScripts(globalName, ["(): any"]);
```

Comment:
Now `$props` doesn't have generics.

https://github.com/sveltejs/svelte/blob/f2eed15c021b038a77cb97e71c565823ef2c16fb/packages/svelte/src/ambient.d.ts#L339

Reply:
But 1 test is failed, so I reverted.

Reply:
Yeah, this is intentionally typed differently than Svelte core.